downloadDirectory/src/main/java/org/reactome/release/downloadDirectory/CreateReleaseTarball.java

```diff
@@ -0,0 +1,99 @@
+package org.reactome.release.downloadDirectory;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.nio.file.StandardCopyOption;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.io.filefilter.DirectoryFileFilter;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.eclipse.jgit.api.Git;
+import org.eclipse.jgit.api.errors.GitAPIException;
+
+public class CreateReleaseTarball {
+	private static final Logger logger = LogManager.getLogger();
+	public static boolean runPerl = true;
+	
+	public static void execute(String releaseNumber, String releaseDownloadDir) throws IOException, InterruptedException {
+		logger.info("Running CreateReleaseTarball step");
+		
+		if (runPerl) {
+			String tarballCommand = "perl " + releaseDownloadDir + "/make_release_tarball.pl " + releaseNumber;
+			Process runPerlTarballCommand = Runtime.getRuntime().exec(tarballCommand);
+			runPerlTarballCommand.waitFor();
+		} else {
+		
+		
+			String tarDirRelease = "reactome_tar/" + releaseNumber + "/Release";
+			String absReleaseDir = "/usr/local/gkb/scripts/release/website_files_update";
+			String absReactomeDir = "/usr/local/reactomes/Reactome/production/Website/static/download/" + releaseNumber;
+			releaseNumber = releaseNumber.replace("/", "");
+			//TODO: When the dust has settled, make sure the tarball contains everything we need
+			
+			// Remove any existing Release repositorys
+			File releaseDir = new File(tarDirRelease);
+			releaseDir.mkdirs();
+			if (releaseDir.list().length > 0) {
+				Process removeReleaseRepo = Runtime.getRuntime().exec("rm -r " + tarDirRelease);
+				removeReleaseRepo.waitFor();
+			} 
+			logger.info("Cloning Release: https://github.com/reactome/Release.git...");
+			try {
+				Git.cloneRepository().setURI("https://github.com/reactome/Release.git").setDirectory(Paths.get(tarDirRelease).toFile()).call();
+				logger.info("Finshed cloning Release repository");
+			} catch (GitAPIException e) {
+				e.printStackTrace();
+			} 
+			logger.info("Copying files into reactome_tar directory...");
+			String tarDirGKB = "reactome_tar/" + releaseNumber + "/reactome/GKB";
+			new File(tarDirGKB).mkdirs();
+			try {
+				FileUtils.copyDirectory(new File(tarDirRelease + "/website"), new File(tarDirGKB));
+			} catch (FileNotFoundException ignore) {}
+			
+			Runtime.getRuntime().exec("rm -f " + tarDirGKB + "/website/html/stats.html");
+			Runtime.getRuntime().exec("rm -f " + tarDirGKB + "/website/html/stats.png");
+			Files.copy(Paths.get(absReleaseDir + "/stats_v" + releaseNumber + ".html"), Paths.get(tarDirGKB + "/website/html/"), StandardCopyOption.REPLACE_EXISTING);
```

Comment:
Could hard-coded paths and partial paths be declared as constants?  They can be re-used and changed easily if in a single place.

Reply:
I've declared the significant majority of paths as final Strings

Reply:
This is in CreateReleaseTarball, which I thought was being "put on hold" or something...

Reply:
Joel said that he left unresolved the comments he wanted me to fix, and since they weren't difficult changes I just decided to do it

Reply:
Well... ok. There were issues in this class RE: making system calls, but I wasn't going to press those issues since this class was not going to be used. Maybe it should just be removed entirely?

Reply:
Sorry, I should've been clearer about the unresolved comments.  For this file, I left them just so they were there if we decided to go ahead with this file later, but I didn't mean that they had to be done right now.

Reply:
Yea I think this is my bad, was just in 'go-getter' mode this morning and went ahead and did it

Reply:
No, that's absolutely fine.  I just should've been clearer that I wasn't meaning to say this had to be done, but no harm that it is and we're ready if we go ahead.